app/javascript/src/components/Dashboard/Contacts/constants.js

```diff
@@ -10,20 +11,13 @@ const DUMMY_CONTACT_DATA = {
 
 export const ROW_DATA = Array(100).fill(DUMMY_CONTACT_DATA);
 
-export const ROLES = [
-  {
-    label: "UX designer",
-    value: "uxDesigner",
-  },
-  {
-    label: "Tester",
-    value: "tester",
-  },
-  {
-    label: "Analyst",
-    value: "analyst",
-  },
-];
+export const ROLES = buildSelectOption([
+  "UX designer",
+  "Tester",
+  "Analyst",
+  "Back-end developer",
+  "Front-end developer",
+]);
```

Comment:
Go through my suggestion once again and see if this is the util name I had suggested. Also try to understand why we need to make that name change here.